polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -443,46 +436,46 @@ mlir::Value MLIRScanner::createAllocOp(mlir::Type t, VarDecl *name,
         // memory space that is not zero, therefore is illegal to create a
         // Memref2Pointer operation that yields a result not in the same memory
         // space.
-        auto memRefToPtr = abuilder.create<polygeist::Memref2PointerOp>(
+        auto memRefToPtr = aBuilder.create<polygeist::Memref2PointerOp>(
             varLoc, LLVM::LLVMPointerType::get(t, 0), alloc);
-        alloc = abuilder.create<polygeist::Pointer2MemrefOp>(
-            varLoc, mlir::MemRefType::get(-1, t, {}, memspace), memRefToPtr);
+        alloc = aBuilder.create<polygeist::Pointer2MemrefOp>(
+            varLoc, MemRefType::get(-1, t, {}, memspace), memRefToPtr);
       }
-      alloc = abuilder.create<mlir::memref::CastOp>(
-          varLoc, mlir::MemRefType::get(-1, t, {}, 0), alloc);
+      alloc = aBuilder.create<memref::CastOp>(
+          varLoc, MemRefType::get(-1, t, {}, 0), alloc);
       LLVM_DEBUG({
         llvm::dbgs() << "MLIRScanner::createAllocOp: created: ";
         alloc.dump();
         llvm::dbgs() << "\n";
       });
 
-      if (t.isa<mlir::IntegerType, mlir::FloatType>()) {
-        mlir::Value idxs[] = {abuilder.create<ConstantIndexOp>(loc, 0)};
-        abuilder.create<mlir::memref::StoreOp>(
-            varLoc, abuilder.create<mlir::LLVM::UndefOp>(varLoc, t), alloc,
-            idxs);
+      if (t.isa<IntegerType, FloatType>()) {
+        Value idxs[] = {aBuilder.create<arith::ConstantIndexOp>(Loc, 0)};
+        aBuilder.create<memref::StoreOp>(
+            varLoc, aBuilder.create<LLVM::UndefOp>(varLoc, t), alloc, idxs);
       }
     }
   } else {
-    auto mt = t.cast<mlir::MemRefType>();
+    auto mt = t.cast<MemRefType>();
     auto shape = std::vector<int64_t>(mt.getShape());
     auto pshape = shape[0];
 
     if (name)
-      if (auto var = dyn_cast<VariableArrayType>(
+      if (auto var = dyn_cast<clang::VariableArrayType>(
```

Comment:
```suggestion
      if (auto Var = dyn_cast<clang::VariableArrayType>(
```

Reply:
I did not modify the case on that file in this PR... figured is long enough but I'll address your latest comments.